src/inventory.cpp

```diff
@@ -1036,11 +1037,19 @@ void inventory::assign_empty_invlet( item &it, const Character &p, const bool fo
         }
     }
     if( cur_inv.count() < inv_chars.size() ) {
+        // XXX YUCK I don't know how else to get the keybindings
```

Comment:
Fixme's are easy to search for.
```suggestion
        // XXX YUCK I don't know how else to get the keybindings
        // FIXME: Find a better way to get bound keys
```